### PR TITLE
tikv,pd: add config of min-resolved-ts

### PR DESCRIPTION
**Pull request by lhy1024**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by rleungx — APPROVED** [review]

**Review by HuSharp — APPROVED** [review]

**Comment by lhy1024** [comment]
@Oreoxmt PTAL

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -989,10 +989,10 @@ raftstore 相关的配置项。
 + 默认值：1MB
 + 最小值：0
 
-### `report-min-resolved-ts-interval`
+### `report-min-resolved-ts-interval` <span class="version-mark">从 v6.0.0 版本开始引入</span>
 
-+ 设置 PD leader 收到 Resolved TS 的最小间隔时间。如果该值设置为 `0`，表示禁用该功能。
-+ 默认值：`"1s"`，即最小正值
++ 设置 PD leader 收到 Resolved TS 的间隔时间。如果该值设置为 `0`，表示禁用该功能。
++ 默认值：`"1s"`，即最小正值(v6.3.0起)；v6.0.0、v6.1.0、v6.2.0 为 "0s"
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:995]:
```suggestion
+ 默认值：在 v6.3.0 之前版本中为 `"0s"`，在 v6.3.0 及之后的版本中为 `"1s"`，即最小正值。
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -159,6 +159,13 @@ pd-server 相关配置项。
 >
 > 如果是从 v4.0 升级至当前版本，升级后的 `flow-round-by-digit` 行为和升级前的 `trace-region-flow` 行为默认保持一致：如果升级前 `trace-region-flow` 为 false，则升级后 `flow-round-by-digit` 为 127；如果升级前 `trace-region-flow` 为 true，则升级后 `flow-round-by-digit` 为 3。
 
+### `min-resolved-ts-persistence-interval` <span class="version-mark">从 v6.0.0 版本开始引入</span>
+
++ 设置 PD leader 对集群中 Resolved TS 最小值进行持久化的间隔时间。如果该值设置为 `0`，表示禁用该功能。
++ 默认值：`"1s"`，即最小正值(v6.3.0起)；v6.0.0、v6.1.0、v6.2.0 为 "0s"
```
Comment (Oreoxmt) [inline pd-configuration-file.md:165]:
```suggestion
+ 默认值：在 v6.3.0 之前版本中为 `"0s"`，在 v6.3.0 及之后的版本中为 `"1s"`，即最小正值。
```

File: pd-configuration-file.md [hunk 3]
```diff
@@ -159,6 +159,17 @@ pd-server 相关配置项。
...
+### `min-resolved-ts-persistence-interval` <span class="version-mark">从 v6.0.0 版本开始引入</span>
+
++ 设置 PD leader 对集群中 Resolved TS 最小值进行持久化的间隔时间。如果该值设置为 `0`，表示禁用该功能。
++ 默认值：在 v6.3.0 之前版本中为 `"0s"`，在 v6.3.0 及之后的版本中为 `"1s"`，即最小正值。
++ 最小值：0
++ 单位：秒
+
+> **注意：**
+>
+> 如果是从 v6.0.0~v6.2.x 升级上来的，`min-resolved-ts-persistence-interval` 将保持此前配置的 `"0s"`，使用该功能的话需要手动开启。
```
Comment (Oreoxmt) [inline pd-configuration-file.md:171]:
```suggestion
> 对于从 v6.0.0~v6.2.0 升级上来的集群，`min-resolved-ts-persistence-interval` 的默认值不发生变化，即为 `"0s"`。若要开启该功能，需要手动修改该配置项的值。
```

v6.2 是 DMR 版本

**Review by Oreoxmt — APPROVED** [review]

File: pd-configuration-file.md [hunk 4]
```diff
@@ -159,6 +159,17 @@ pd-server 相关配置项。
...
+### `min-resolved-ts-persistence-interval` <span class="version-mark">从 v6.0.0 版本开始引入</span>
+
++ 设置 PD leader 对集群中 Resolved TS 最小值进行持久化的间隔时间。如果该值设置为 `0`，表示禁用该功能。
++ 默认值：在 v6.3.0 之前版本中为 `"0s"`，在 v6.3.0 及之后的版本中为 `"1s"`，即最小正值。
++ 最小值：0
++ 单位：秒
+
+> **注意：**
+>
+> 对于从 v6.0.0~v6.2.0 升级上来的集群，`min-resolved-ts-persistence-interval` 的默认值不发生变化，即为 `"0s"`。若要开启该功能，需要手动修改该配置项的值。
```
Comment (qiancai) [inline pd-configuration-file.md:171]:
```suggestion
> 对于从 v6.0.0~v6.2.0 升级上来的集群，`min-resolved-ts-persistence-interval` 的默认值在升级后将不会发生变化，即仍然为 `"0s"`。若要开启该功能，需要手动修改该配置项的值。
```

**Comment by Oreoxmt** [comment]
@qiancai PTAL again

**Review by qiancai — APPROVED** [review]
/lgtm

**Pull request merged**